### DISCUSSION TITLE
Calling another .py from the converted .exe file

### DISCUSSION THREAD

**Discussion by ntahlahweh** [brentvollebregt/auto-py-to-exe#499] [Q&A] [answered]
Hi everyone, I just want to ask how can I call another `.py` from the converted `.exe` file?

I have a main python code called `main_2.py` file that act as the main interface for program that I created which in this case is the one that will be converted into `.exe`. One of its main function is to open up another python code called `analysis.py` which in this case is another GUI itself that will do other task.  The order of the file could be seen from the directory here:

![Screenshot 2024-09-16 130943](https://github.com/user-attachments/assets/076b7bc7-79e4-4e0b-9df0-c98a389a022c)

My question is can the converted `.exe` file called the `.py` directly or should I convert the `analysis.py` into `.exe` as well?

Thank you

**Comment by brentvollebregt** [comment, marked as answer]
Select your "entrypoint" file - the tool will look for `import` statements and include those files and libraries too (where possible). I go over this in the help post linked in the README and tool - under ["How do I Convert All My Python Files?"](https://nitratine.net/blog/post/issues-when-using-auto-py-to-exe/#how-do-i-convert-all-my-python-files)

**Reply by ntahlahweh** [reply]
Sorry for the late reply, after several trial and error I finally figured out what you mean and how it works. basically I just have to make sure the location of the .py code is properly call in order for it to works! 

But the other way around I did is to convert that `.py` code into `.exe` to call it and it works too.

Thanks!
